org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/DebugException.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug;
+
+public class DebugException extends Exception {
+    private static final long serialVersionUID = 1L;
+
+    public DebugException() {

```

Comment:
DebugException extends Exception. And from javadoc of Exception's default constructor `java.lang.Exception.Exception()`, it said `
Constructs a new exception with null as its detail message. The cause is not initialized, and may subsequently be initialized by a call to initCause.
`